package.json

```diff
@@ -2,8 +2,8 @@
   "name": "recommendations-impact-dashboard",
   "scripts": {
     "test": "ts-node node_modules/jasmine/bin/jasmine src/**/*.spec.ts",
-    "build": "ng build && cp -r src/main/angular/WEB-INF src/main/webapp/WEB-INF && mvn package appengine:run -DskipTests",
-    "deploy": "ng build && cp -r src/main/angular/WEB-INF src/main/webapp/WEB-INF && mvn package appengine:deploy -DskipTests",
+    "build": "ng build && cp -r src/main/angular/WEB-INF src/main/webapp/WEB-INF && mvn clean && mvn package appengine:run -DskipTests",
+    "deploy": "ng build && cp -r src/main/angular/WEB-INF src/main/webapp/WEB-INF && mvn clean && mvn package appengine:deploy -DskipTests",
```

Comment:
Re: comment above, I DO think it makes sense to have mvn clean in a deploy rule.